src/components/app/Scores/index.tsx

```diff
@@ -1,104 +1,96 @@
 import { Grouping } from '../../../types/grouping';
 import { GenericList, GenericListItem } from '../GenericList';
-import { HeadCell } from '../GenericList/HeadCell';
 import _ from 'lodash';
-import { getAverage, getMedian } from '../../../utils/commonUtils';
+import {
+    amountToMMs,
+    getAverage,
+    getMedian,
+    getTvlImpact,
+} from '../../../utils/commonUtils';
 import { ScoreRowCollapse } from '../../common/ScoreRowCollapse';
-import { Semaphore } from '../../common/Semaphore';
-import { Link } from 'react-router-dom';
-import MonetizationOnRoundedIcon from '@material-ui/icons/MonetizationOnRounded';
+import { getStrategyTVLsPerProtocol } from '../../../utils/strategiesHelper';
+import { headCells } from './headerDefinition';
 
-const headCells: HeadCell[] = [
-    {
-        numeric: false,
-        disablePadding: false,
-        label: 'TVL Link',
-        align: 'center',
-        format: (item: GenericListItem) => {
-            return (
-                <>
-                    <Link
-                        color="inherit"
-                        target="_blank"
-                        to={`/query/${item.groupingId}/group/${item.label}`}
-                    >
-                        <Tooltip
-                            title={`View how the total TVL is allocated in the group ${item.label}`}
-                        >
-                            <MonetizationOnRoundedIcon
-                                fontSize="small"
-                                htmlColor="gray"
-                            />
-                        </Tooltip>
-                    </Link>
-                </>
-            );
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            width: '100%',
+            padding: theme.spacing(2),
+            textAlign: 'center',
         },
-    },
-    {
-        id: 'label',
-        numeric: false,
-        disablePadding: false,
-        label: 'Group',
-        align: 'center',
-    },
-    {
-        id: 'averageScore',
-        numeric: true,
-        disablePadding: false,
-        label: 'Average',
-        align: 'center',
-        format: (item: GenericListItem, value: string | number | boolean) => {
-            return parseFloat(value.toString()).toFixed(3);
-        },
-    },
-    {
-        id: 'medianScore',
-        numeric: true,
-        disablePadding: false,
-        label: 'Median',
-        align: 'center',
-        format: (item: GenericListItem, value: string | number | boolean) => {
-            return parseFloat(value.toString()).toFixed(3);
-        },
-    },
-];
+    })
+);
 
 export const Scores = () => {
     const [groups, setGroups] = useState<Grouping[]>([]);
+    const [items, setItems] = useState<any[]>([]);
+    const [isLoading, setIsLoading] = useState<boolean>(false);
     const { groupings } = initFB();
     const groupingId = 'default';
     const [groupData, { loading, error }] = useOnGet(groupings, groupingId);
```

Comment:
i think we can move this to useEffect 

https://reactjs.org/docs/hooks-effect.html

Reply:
As `useOnGet` is a hook, it is not possible. The hooks can only be called inside of the body of a function component.